CHANGELOG.md

```diff
@@ -1,12 +1,28 @@
 Here we document changes that affect the public API or changes that needs to be communicated to other developers. 
 
+## 2020-04-03 OrdinalPropertyState
+Added a OrdinalPropertyState helper for constructing ordinal properties.
+And a factory function `util::ordinalColor` for OrdinalProperties representing Colors
+When instantiating a Ordinal Property for a color value one would need to write something along
+there lines
+```c++
+color("cubeColor", "Cube Color", vec4(0.11f, 0.42f, 0.63f, 1.0f),
+     {vec4(0.0f), ConstraintBehavior::Immutable},
+     {vec4(1.0f), ConstraintBehavior::Immutable}, vec4(0.01f),
+     InvalidationLevel::InvalidOutput, PropertySemantics::Color)
+```
+by using the helper function most of the boilerplate can be removed:
+```c++
+color{"cubeColor", "Cube Color", util::ordinalColor(0.11f, 0.42f, 0.63f)}
```

Comment:
Nice!

Reply:
thanks, like this we could also do color{"cubeColor", "Cube Color", colors::blue) for example

Reply:
I like that, but the name `colors::blue` might be ambigious with `vec4(0,0,1,1)`, maybe `ordinal::color::blue` or something similar